src/pages/NotFound/NotFound.test.js

```diff
@@ -1,10 +1,24 @@
 import React from "react";
-import { shallow } from "enzyme";
+import configureStore from "redux-mock-store";
+import { Provider } from "react-redux";
+import { mount } from "enzyme";
+import { MemoryRouter } from "react-router";
+import dataDump from "testing/complete-redux-store-dump";
+
 import NotFound from "./NotFound";
 
+const mockStore = configureStore([]);
+
 describe("NotFound page", () => {
-  it("renders without crashing and matches snapshot", () => {
-    const wrapper = shallow(<NotFound />);
-    expect(wrapper).toMatchSnapshot();
+  it("should display when unknown route is accessed", () => {
+    const store = mockStore(dataDump);
+    const wrapper = mount(
+      <Provider store={store}>
+        <MemoryRouter initialEntries={["/foobar11"]}>
+          <NotFound />
```

Comment:
This test isn't actually testing anything, it's just rendering the `NotFound` component. You'll want to use the `TestRoute` component. You can see it in action in the `InfoPanel` or `ModelDetail` tests.